airbyte-integrations/connector-templates/connector_acceptance_test_files/acceptance-test-config.yml.hbs

```diff
@@ -3,8 +3,6 @@
 connector_image: {{ connectorImage }}
 acceptance_tests:
   spec:
-    tests:
```

Comment:
Unless there's been a deviation in how CAT works orI'm missing context, just deleting this would cause us to not run this specific test which isn't quite what we want. Python connectors probably still need this as it is used now. And low-code would instead need to know how to point to parsing through the manifest to a specific part.

I think this requires a change to the underlying CAT framework

Reply:
So ironically what you're saying is we're either not running a CAT test on spec for all manifest-based connectors that don't have a separate spec file, or it somehow runs mysteriously and we're not sure exactly how? 🤦🏼